src/Http/WebUtilities/src/IBufferedWriteStream.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.WebUtilities
+{
+    public interface IBufferedWriteStream : IAsyncDisposable, IDisposable
```

Comment:
Marking this as `IDisposable` helps in situations like this one:

https://github.com/aspnet/AspNetCore/blob/d6c629a005a859ca653cbb2ec96d4b6c385584cf/src/Mvc/Mvc.ViewFeatures/src/ViewComponentResultExecutor.cs#L157

And it also would simplify this:

https://github.com/aspnet/AspNetCore/blob/d6c629a005a859ca653cbb2ec96d4b6c385584cf/src/Mvc/Mvc.NewtonsoftJson/src/NewtonsoftJsonOutputFormatter.cs#L169

What do you think?

Reply:
Looks good.